config/settings.yml

```diff
@@ -443,6 +443,13 @@ va_mobile:
   key_path: /fake/client/key/path
   timeout: 15
 
+lighthouse:
+  api_key: fake_key
```

Comment:
The idea here is that we can share this API key across any services that leverage Lighthouse APIs.